hack/python-sdk/gen-sdk.sh

```diff
@@ -37,31 +35,40 @@ fi
 echo "Generating OpenAPI specification ..."
 echo "./hack/update-codegen.sh already help us generate openapi specs ..."
 
-if [[ ! -f "$SWAGGER_CODEGEN_JAR" ]]; then
-  echo "Downloading the swagger-codegen JAR package ..."
-  wget -O "${SWAGGER_CODEGEN_JAR}" ${SWAGGER_JAR_URL}
-fi
-
-if [ -z `which java` ]; then
-  apt-get update -y
-  # TODO: update this fallback (remove it) when openjdk-11 completely deprecated
-  echo "Installing OpenJDK 11 with fallback to OpenJDK 17"
-  apt-get install -y openjdk-11-jdk || apt-get install -y openjdk-17-jdk
-fi
-
 echo "Generating swagger file ..."
 go run "${repo_root}"/hack/swagger/main.go ${VERSION} >"${SWAGGER_CODEGEN_FILE}"
 
 echo "Removing previously generated files ..."
 rm -rf "${SDK_OUTPUT_PATH}"/docs/V1*.md "${SDK_OUTPUT_PATH}"/jobset/models "${SDK_OUTPUT_PATH}"/test/test_*.py
 
 echo "Generating Python SDK for JobSet..."
-java -jar "${SWAGGER_CODEGEN_JAR}" generate -i ${SWAGGER_CODEGEN_FILE} -g python -o "${SDK_OUTPUT_PATH}" -c "${SWAGGER_CODEGEN_CONF}"
+
+# Defaults the container engine to docker
+CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}
```

Comment:
AFAIK this works with current CI so no need for https://github.com/kubernetes/test-infra/pull/33955

Reply:
I just read the comments on test-infra, I will close the pr and update the `pull-jobset-verify-main` in config to include 

```
 securityContext:
            privileged: true
```
for docker-in-docker.

Sorry for the confusion!